plothist/variable_registry.py

```diff
@@ -277,15 +277,15 @@ def update_variable_registry_ranges(
 
     for variable_key in variable_keys:
         variable = get_variable_from_registry(variable_key, path=path)
-        if "bins" not in variable.keys() or "range" not in variable.keys():
+        if not all(key in variable.keys() for key in ["bins", "range", "name"]):
             raise RuntimeError(
-                f"Variable {variable_key} does not have a bins or range property in the registry {path}."
+                f"Variable {variable_key} does not have a name, bins and/or range property in the registry {path}."
```

Comment:
I do not understand the and/or. The error is thrown if the variable is missing a name, bin or range field.